hmt_escrow_js/server.js

```diff
@@ -0,0 +1,60 @@
+require('dotenv').config()
```

Comment:
We probably don't need the server file in here do we?

Reply:
removed, you're right